### DISCUSSION TITLE
(idea, not necessarily ntsc-rs related) open-source rust film emulation plugin?

### DISCUSSION THREAD

**Discussion by dawnch0rus** [ntsc-rs/ntsc-rs#116] [Ideas]
something that i think would be an interesting side project to ntsc-rs is a video effect that emulates the distortions of movie film, particularly 35mm. i've looked at a lot of the options out there and nearly all of them have really big drawbacks. there are some decent options like dehancer or various effect stacks that somewhat get you there, and some of the lut packs out there are free and licensed very generously. the downsides are that dehancer is expensive, and the lut packs lack the options necessary to replicate the texture and grain of real film.

i think there's a real niche here that could be filled. a film simulation tool that's open source, has some of the more in-depth simulation options of something like dehancer, and supports the same variety of applications as ntsc-rs. 

considering how accurate ntsc-rs is, i think it'd be fairly feasible. especially considering that film simulation is something that a lot of people have documented their work in (notably the outstanding [steve yedlin](https://www.yedlin.net/NerdyFilmTechStuff/)), i would imagine it would be fairly simple to get a start in recreating some of these effects.

**Comment by Degamisu** [comment]
Holy! I was literally about to ask for this. I was thinking it would definitely have more than just 35mm Movie, it should have picture modes aswell. I actually have an app called 8mm and it does surprisingly well at emulating it. its free.

**Comment by oisintheblue** [comment]
this would be epic if this happened.
(offtopic part of the comment, but honestly if valadaptive ever gets around to recreating vhs type audio, it should be its own separate thing since ntsc-rs is mostly supposed to be a plugin for video editors, as it may not be possible to have video and audio effects in one plugin.)

**Comment by valadaptive** [comment]
> if valadaptive ever gets around to recreating vhs type audio

If you want VHS audio, you can probably just use [ChowTape](https://github.com/jatinchowdhury18/AnalogTapeModel)

**Reply by Degamisu** [reply]
ah yes. I use chow for a lot of analog audio tape ones, its a good pick. but ive never seen it used for VHS? i might make a preset for that.

**Reply by oisintheblue** [reply]
yeah, but it only has the basic stuff for vhs audio.

**Reply by Degamisu** [reply]
> yeah, but it only has the basic stuff for vhs audio.

Not necessarily, it has something called `Bias` which when lowered, creates a "deadzone" effect.

**Reply by fuki98** [reply]
Looks sick! Had no idea something like this existed.

**Reply by PacificGM** [reply]
I use [BPB Dirty VHS](https://bedroomproducersblog.com/2023/11/20/bpb-dirty-vhs/) for VHS audio but I can't say for sure on how accurate it is. They did use real VHS tapes but they may have used PAL tapes.

**Comment by Henrycounty** [comment]
I don't know how much has been researched when it comes to film emulation, but i like this idea 💡. I always felt like the grain and those little sparkles of dust gives a nice texture that digital video can't do. I wish that someone does at least an aproximation of that.
